app/src/main/java/org/americanairlines/our1grouprestaurantmapproject/util/DebugLogger.kt

```diff
@@ -0,0 +1,18 @@
+package org.americanairlines.our1grouprestaurantmapproject.util
+
+import android.util.Log
+
+class DebugLogger {
+    companion object {
+        private const val LOG = "LOG_X"
+
+        fun log(message: String) {
+            Log.d(LOG, message)
+        }
+
+        fun elog(errMessage: String) {
+            Log.d(LOG, errMessage)
```

Comment:
Functionally the same as log above. Should use log.e so that it gets red highlighting in logcat, or so that it can be filtered on ERROR instead of DEBUG.